manifests/concourse-manifest/concourse-base.yml

```diff
@@ -73,21 +73,22 @@ jobs:
           databases:
             - name: atc
               role: atc
-              password: dummy-password
+              password: (( grab secrets.concourse_postgres_password ))
```

Comment:
I guess that's specific to our postgres job that it doesn't require PW when connecting via localhost. We could argue that generating a password reduces attack surface. In the future, we might decide to make that posgres job safer or perhaps have some different setup where ATC is connecting on eth0 IP (or e.g. external workers need to connect to DB). 

This way or another, generating the password doesn't hurt anything so I think we should keep it.